src/main/antlr4/Manifold.g4

```diff
@@ -12,8 +16,9 @@ WHITESPACE: [ \t\r\n]+ -> skip;
 
 INTEGER_VALUE: [0-9]+;
 BOOLEAN_VALUE: 'false' | 'true';
+TYPE_KEYWORD: 'Type';
 
-tupleTypeValueEntry: (IDENTIFIER ':')? typevalue (':' expression)?;
+tupleTypeValueEntry: (IDENTIFIER ':')? typevalue ('=' expression)?;
```

Comment:
Can you comment on why this was changed from `:` to `=`? I can't remember if we talked about this but I feel as though we did -- help refresh my memory?